cfme/tests/test_replication.py

```diff
@@ -198,7 +198,38 @@ def test_replication_appliance_set_type_global_ui():
             1.
             2. No error, appliance subscribed.
     """
-    pass
+    remote_app, global_app = configured_appliance, unconfigured_appliance
+    app_params = dict(
+        region=99,
+        dbhostname='localhost',
+        username=credentials["database"]["username"],
+        password=credentials["database"]["password"],
+        dbname='vmdb_production',
+        dbdisk=global_app.unpartitioned_disks[0],
+        fetch_key=remote_app.hostname,
+        sshlogin=credentials["ssh"]["username"],
+        sshpass=credentials["ssh"]["password"],
+    )
+
+    global_app.appliance_console_cli.configure_appliance_internal_fetch_key(**app_params)
+    global_app.evmserverd.wait_for_running()
+    global_app.wait_for_web_ui()
+
+    # Making configured app to Remote Appliance using UI
+    remote_region = remote_app.collections.regions.instantiate()
```

Comment:
Will you need to specify the region number here?

Reply:
The default value is 0 for region and I am also expecting the same so no need to provide value like below:
 `remote_region = remote_app.collections.regions.instantiate(number=0)`